### PR TITLE
Add icon to make the order book more visible

### PR DESCRIPTION
**Pull request by anxolin**
Just a test addressing comment: https://gnosisinc.slack.com/archives/CSGLUC1BN/p1586443057005800

"I like the pop up but it took me some time to find the button"

image

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr868--dexreact.review.gnosisdev.com

File: src/components/OrderBookBtn.tsx [hunk 1]
```diff
@@ -10,6 +10,19 @@ import TokenSelector from './TokenSelector'
...
+import { faChartLine } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+const ViewOrderBookBtn = styled.button`
+  margin: 0 0 0 auto;
+  text-align: right;
+
+  svg {
+    font-size: 1.7rem;
+    color: var(--color-text-active);
```
Comment (W3stside) [inline src/components/OrderBookBtn.tsx:22]:
```suggestion
    fill: var(--color-text-active);
```

**Comment by anxolin** [comment]
> keep in mind it still looks bad on dark mode!

I see it like this. Or you mean the actual order book?

image

**Comment by alfetopito** [comment]
> > keep in mind it still looks bad on dark mode!
> 
> I see it like this. Or you mean the actual order book?
> 
> image

Yes, I meant the orderbook itself.
Just raising the fact whenever someone touches the orderbook in hopes someone will notice :)

The axis text is almost unreadable
screenshot_2020-04-10_07-58-31

**Comment by anxolin** [comment]
> Yes, I meant the orderbook itself. Just raising the fact whenever someone touches the orderbook in hopes someone will notice :)

I see. Yes, the styles in darkmode are a bit bad, but is not related to this PR. I'll make an issue for this.

**Pull request merged**